cores/esp32/esp32-hal-uart.c

```diff
@@ -66,27 +66,41 @@ static uart_t _uart_bus_array[3] = {
 };
 #endif
 
+static void _uart_rx_flush( uart_t * uart){
+    uint8_t c;
+    UART_MUTEX_LOCK();
+    UBaseType_t spaces=0;
+    if(uart->queue != NULL) {
+        spaces=uxQueueSpacesAvailable( uart->queue);
+    }
+    while(((spaces--) >0)&&(uart->dev->status.rxfifo_cnt || (uart->dev->mem_rx_status.wr_addr != uart->dev->mem_rx_status.rd_addr))) {
+        c = uart->dev->fifo.rw_byte;
+        xQueueSend(uart->queue, &c,0);
+    }
+	UART_MUTEX_UNLOCK();
```

Comment:
looks like an extra tab/space is here as well

Reply:
your correct, another TAB, Maybe fixed ;)